metricbeat/docs/modules/mysql.asciidoc

```diff
@@ -101,6 +101,8 @@ metricbeat.modules:
   
   # Client certificate key file
   #ssl.key: "/etc/pki/client/cert.key"----
```

Comment:
```suggestion
  #ssl.key: "/etc/pki/client/cert.key"
```

I think the actual problem was that the text was pasted without a new line character, hence the trailing dashes.

Unfortunately, it's not enough to just make this change, we need to run `make update` to change all the generated files too. I'll do that in a moment.

Reply:
Nice catch! I didn't see this part.